test-run.py

```diff
@@ -272,6 +272,9 @@ def open_as_utf8(*args, **kwargs):
 
     status = 0
 
+    # set allocator for memtx, default small
+    os.environ['TEST_RUN_MEMTX_ALLOCATOR'] = Options().args.allocator
```

Comment:
The similar `REPLICATION_SYNC_TIMEOUT` is set in `lib/__init.py`, I would do the same for this environment variable.

Reply:
Moved.